src/pages/journal/components/test-outcome/test-outcome.ts

```diff
@@ -46,6 +46,16 @@ export class TestOutcomeComponent {
     this.navController.push('WaitingRoomPage');
   }
 
+  writeUpTest() {
+    this.store$.dispatch(new ActivateTest(this.slotId));
+    this.navController.push('OfficePage');
+  }
+
+  resumeTest() {
```

Comment:
Appreciate the wider concern in covering the resume use case. I hadn't thought of this.